test/box-luatest/error_subsystem_improvements_test.lua

```diff
@@ -1,6 +1,22 @@
 local t = require('luatest')
 local g = t.group()
 
+-- Test the `prev' argument to the table constructor of `box.error.new'
+-- (gh-9103).
+g.test_error_new_prev_arg = function()
+    local e1 = box.error.new({"errare"})
```

Comment:
Nit: just for your knowledge, you can omit parentheses for table constructors.

Reply:
Omitting parentheses in such cases is discouraged by our [Lua style guide](https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#idioms-and-patterns).

Reply:
Didn't know about that, okay.